features/environment.py

```diff
@@ -22,25 +22,33 @@ class UAClientBehaveConfig:
         A string with an image name that should be used instead of building a
         fresh image for this test run.  If specified, image_clean will be set
         to False.
+    :param destroy_instances:
+        This boolean indicates that test containers should be destroyed after
+        the completion. Set to 0 to leave instances running.
```

Comment:
This is Python-level documentation, so this should be:

```suggestion
        the completion. Set to False to leave instances running.
```